SWLOR.Game.Server/SWLOR.Game.Server.csproj

```diff
@@ -1648,7 +1652,7 @@
   </ItemGroup>
   <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
   <PropertyGroup>
-    <PostBuildEvent>xcopy "$(TargetDir)*.*" "C:/NeverwinterNights/NWN/mono" /E /Y /C
+    <PostBuildEvent>xcopy "$(TargetDir)*.*" "C:\Users\jesse\Documents\Neverwinter Nights\mono\" /E /Y /C
```

Comment:
If you don't mind changing this back for the pull request I'd appreciate it. I'm going to update the location to use the default NWN directory.